python/cuml/cuml/fil/fil.pyx

```diff
@@ -557,15 +557,15 @@ cdef class ForestInference_impl():
     def __dealloc__(self):
         cdef handle_t* handle_ = <handle_t*><size_t>self.handle.getHandle()
         fil_dtype = self.get_dtype()
```

Comment:
This line was causing troubles, as modules may have been unloaded by the Python interpreter by the time the `__dealloc__` method is called. When I inserted a `print(np)` statement here, it yielded `None`.